contracts/examples/IncentivisedSlidingWindowOracle.sol

```diff
@@ -0,0 +1,176 @@
+pragma solidity =0.6.6;
+
+import '@uniswap/v2-core/contracts/interfaces/IUniswapV2Factory.sol';
+import '@uniswap/v2-core/contracts/interfaces/IUniswapV2Pair.sol';
+import '@uniswap/lib/contracts/libraries/FixedPoint.sol';
+
+import '../libraries/SafeMath.sol';
+import '../interfaces/IERC20.sol';
```

Comment:
What you think of using `@openzeppelin/contracts/token/ERC20/SafeERC20.sol` and then we use `safeTransfer`  for token transfers instead.